### PR TITLE
Migrate Lidar from STM32 to Raspberry Pi

### PR DESCRIPTION
**Pull request by ecourtois**

### CODE REVIEW

**Review by gdoffe — CHANGES_REQUESTED** [review]
Good job ! A new important step !

A small fix needed before approval.

File: platforms/pegasus/trace_utils.cpp
Comment (ecourtois) [reply platforms/pegasus/trace_utils.cpp:11]: ZTIMER_USEC is used in all other threads, Is there a reason to use MSEC here?

Comment (gdoffe) [reply platforms/pegasus/trace_utils.cpp:11]: Yes we are using milliseconds => MSEC.
This should be done this way in all other threads.

I will fix all remaining USEC threads soon, however as you are modifying this one, good occasion to use MSEC now.

File: platforms/pf_test/trace_utils.cpp
Comment (gdoffe) [reply platforms/pf_test/trace_utils.cpp:14]: I'm ok with this!

File: platforms/pegasus/trace_utils.cpp
Comment (ecourtois) [reply platforms/pegasus/trace_utils.cpp]: Ok! Done!

**Review by gdoffe — APPROVED** [review]
LGTM !

**Pull request merged**
